### PR TITLE
Make crux-visit-term-buffer ask to restart dead process

### PR DESCRIPTION
**Pull request by justbur**

### CODE REVIEW

**Comment by justbur** [comment]
not working yet

File: crux.el [hunk 1]
```diff
@@ -120,11 +120,16 @@ the current buffer."
...
-  "Create or visit a terminal buffer."
+  "Create or visit a terminal buffer.
+If the process in that buffer died, ask to restart."
   (interactive)
   (crux-start-or-switch-to (lambda ()
                              (ansi-term crux-shell (concat crux-term-buffer-name "-term")))
-                           (format "*%s-term*" crux-term-buffer-name)))
+                           (format "*%s-term*" crux-term-buffer-name))
+  (when (and (null (get-buffer-process (current-buffer)))
+             (y-or-n-p "Restart? "))
```
Comment (bbatsov) [inline crux.el]: I think this message should be a bit more informative.

Comment (justbur) [inline crux.el]: Agreed. See the next commit

**Comment by bbatsov** [comment]
👍

**Pull request merged**